modules/installing-rhv-setting-up-installation-machine.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installing-rhv-setting-up-installation-machine_{context}"]
+= Setting up the installation machine
+
+
+To run the binary `openshift-install` installation program and Ansible scripts, use either the {rh-virtualization} Manager or an Enterprise Linux computer with network access to the {rh-virtualization} environment and the REST API on the {rh-virtualization} Manager/oVirt Engine.
+
+
+.Procedure
+
+. Update or install python3 and ansible. For example:
++
+[source,terminal]
+----
+# dnf update python3 ansible
```

Comment:
We usually use `yum` commands instead of `dnf` commands.

Reply:
DNF is preferred for RHEL 8: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index#notable-changes-to-the-yum-stack_software-management